fixtures/tracks/animal/exercises/TRACK_HINTS.md

```diff
@@ -0,0 +1 @@
+This is the content of the track hints file
```

Comment:
Thank you! My other option was to revert this and re-apply #12. But I prefer having both.